packages/web-react/src/components/Pagination/UncontrolledPagination.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from 'react';
+import { SpiritUncontrolledPaginationProps } from '../../types';
+import Pagination from './Pagination';
+import PaginationItem from './PaginationItem';
+import PaginationLink from './PaginationLink';
+import PaginationLinkPrevious from './PaginationLinkPrevious';
+import PaginationLinkNext from './PaginationLinkNext';
+
+export const UncontrolledPagination = (props: SpiritUncontrolledPaginationProps) => {
```

Comment:
How is the change of the page handled? I think we forgot to implement this: https://github.com/lmc-eu/spirit-design-system/pull/738/files#diff-3f28c4c93ea39848b38622ddfe1400a8dbd31ef9297edc9a330df0081032249c

Now the component is a dump and does not have any state machine. I am expecting from this component that when I click on a page link or an arrow button, it will change the state.

Think about how you will use this component in an app and would do you expect that will do :-)

Reply:
I dealt with exactly what was in the [assignment](https://jira.lmc.cz/browse/DS-329) and there it is said that the component should be stupid and each product will deal with clicking separately.

So I created the maximum with it ready for everyone to supply the logic there themselves, I showed that in the live demo

Reply:
But I could make a similar hook and then use it in that live demo

Reply:
Ok, I think we should move this into a separate branch and PR and discuss the assignment. The Pagination component itself is looking good and it is very close to merge. Our debate around the uncontrolled component can block this. My expectation of would the uncontrolled component goes beyond the original assignment. So let's move on and prepare the issue for this and solve this separately, please.

Reply:
I have already prepared a hook for the basic control and connected it to that demo